pages.pt_BR/common/fd.md

```diff
@@ -0,0 +1,29 @@
+# fd
+
+> Uma alternativa para `find`.
+> Visa ser mais rápido e fácil de usar do que `find`.
+> Mais informações: <https://github.com/sharkdp/fd>.
+
+- Encontra recursivamente arquivos que correspondam ao padrão fornecido no diretório atual:
+
+`fd {{padrão}}`
+
+- Encontra arquivos que começam com "foo":
+
+`fd {{'^foo'}}`
+
+- Encontra arquivos com uma extensão específica:
+
+`fd --extension {{txt}}`
+
+- Encontra arquivos em um diretório específico:
+
+`fd {{padrão}} {{caminho/para/diretório}}`
+
+- Inclui arquivos ignorados e ocultos na pesquisa:
+
+`fd --hidden --no-ignore {{padrão}}`
+
+- Executa um comando em cada resultado de pesquisa retornado:
+
+`fd {{padrão}} --exec {{comando}}`
```

Comment:
The following code fix could be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/fd.md) page is refreshed:

```suggestion
`fd '{{padrão}}' --exec {{comando}}`
```

- Patterns should be surrounded with quotes not to interfere with globbing.
- It's better provide a concrete example instead of generic placeholder for command.
- Our style guide doesn't specify whether this fix should be applied or not.